rest-client/swagger.json

```diff
@@ -162,6 +162,21 @@
       "in": "query",
```

Comment:
Zach, all the comments on this file really apply to the BridgeDoc *.yml files. Be sure to update those and submit another PR to update those, as well as fixing them there. Thanks!